src/main.rs

```diff
@@ -113,8 +120,12 @@ fn exec() -> anyhow::Result<()> {
                     if content_type.starts_with("text/html") {
                         data = minify::html::minify(&data);
                     }
-                    send_result(path_info, data, content_type, status_opt);
-
+                    
+                    if gzip_encoding {
```

Comment:
If I am reading the docs right (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding), I think Bartholomew also must check the `HTTP_ACCEPT_ENCODING` header to see if the client supports `gzip`.

Reply:
Sorry... I meant the environment variable. The header Accept-Encoding becomes the env var HTTP_ACCEPT_ENCODING.

Reply:
thanks. I am now checking for the accept-encoding header before enabling gzip response.